polaris-react/src/components/Banner/Banner.scss

```diff
@@ -1,67 +1,54 @@
 @import '../../styles/common';
```

Comment:
refactoring this a bit to use `border` instead of `box-shadow` for borders, and to get rid of the `--pc` values for background and border since they are not really useful and also cause stylelint failures